src/openms/source/FORMAT/FeatureXMLFile.cpp

```diff
@@ -488,6 +491,7 @@ namespace OpenMS
       {
         map_->setUniqueId(unique_id);
       }
+      last_meta_ = map_;
```

Comment:
All meta values in the first element of the FeatureXML get attached to the map_. last_meta_ is adapted to the current level in the XML e.g. all meta values further down the XML tree (e.g. PepitdeIdentifications, Features etc.) are added at the respective.